src/core/tests/config-test.ini

```diff
@@ -2,14 +2,15 @@
 run = true
 count = 1
 network.client.RPC_CHANNEL_TCP = dsn::tools::asio_network_provider, 65536
-network.client.RPC_CHANNEL_UDP = dsn::tools::asio_network_provider, 65536
+network.client.RPC_CHANNEL_UDP = dsn::tools::asio_udp_provider, 65536
```

Comment:
Most configs do not specify a network provider. It only happens to nativerun_sim_nets and this config-test.